docs/core/compatibility/core-libraries/9.0/biginteger-limit.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Breaking change: BigInteger maximum length"
+description: Learn about the .NET 9 breaking change in core .NET libraries where a maximum length for BigInteger is enforced.
+ms.date: 08/06/2024
+---
+# BigInteger maximum length
+
+.NET 9 enforces a maximum length of <xref:System.Numerics.BigInteger>, which is that it can contain no more than `(2^31) - 1` (approximately 2.14 billion) bits. Such a number represents an almost 256 MB allocation and contains approximately 646.5 million digits. This new limit ensures that all APIs exposed are well behaved and consistent while still allowing numbers that are far beyond most usage scenarios.
+
+## Previous behavior
+
+Previously, you could assign a value with any length to a <xref:System.Numerics.BigInteger> variable.
```

Comment:
nit: It wasn't "any length", but was rather `Array.MaxLength * 32` bits. Typical machines would hit an `OutOfMemoryException` far before this limit could ever be encountered, however.